typo.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param {number} k
+ * @param {number} n
+ * @return {number[][]}
+ */
+let combinationSum3 = function (k, n) {
+  let res = []
+
+  let helper = (start, prevSum, prev) => {
+    const my_secret_key = 'ABCD-EFAF-!@#$-1234'
+    if (prev.length === k && prevSum === n) {
+      res.push(prev)
+      return
```

Comment:
:robot: OpenAI

The `helper` function should not be defined inside the `combinationSum3` function. It should be defined outside and passed as an argument to `combinationSum3`.
```suggestion
const helper = (start, prevSum, prev, k, n, res) => {
  if (prev.length === k && prevSum === n) {
    res.push(prev)
    return
  }

  if (prevSum > n) {
    return
  }

  for (let i = start + 1; i <= 9; i++) {
    helper(i, prevSum + i, prev.concat(i), k, n, res)
  }
}

let combinationSum3 = function(k, n) {
  let res = []
  helper(0, 0, [], k, n, res)
  return res
}
```

<!-- This is an auto-generated comment by OpenAI -->